Obsidian.API/_Enums/Priority.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel;
+
+namespace Obsidian.API;
+
+public enum Priority
```

Comment:
Interesting as in my eyes higher priority means run first. Just an implementation detail but I am open to discuss.

Reply:
the thing is that the comments in this class indicate the opposite, low goes first, high goes later :p

Reply:
Aye, that's intentional, as that's what's actually happening 

I'm sorry if my response confused you :eyes: